### ISSUE TITLE
Form attributes are being added with extra quotes

### ISSUE THREAD

**Issue by juangadiel** [mautic/mautic#13715]
### Mautic Version

5.0.x series

### Way of installing

Other (please specify if relevant in the bug report)

### PHP version

8.2.18

### What browsers are you seeing the problem on?

Not relevant

### What happened?

**Way of installing**: Gitpod from official Mautic repo on branch 5.x
**Browser**: Brave

When I add attributes (class="custom-class") to a form, they get rendered with double quotes in the HTML.

![image](https://github.com/mautic/mautic/assets/706997/1583317b-779f-49c0-b295-9ba034a3280a)

![image](https://github.com/mautic/mautic/assets/706997/b0a008a1-daf9-44a9-be4d-f05a00e99976)

This behavior is happening to both Campaign and Standalone forms.

### How can we reproduce this issue?

1. Open a new Gitpod from https://github.com/mautic/mautic on branch 5.x
2. Log in to Mautic
3. Navigate to Components > Forms, then create a new Form (Campaign or Standalone)
4. Enter a name for the form
5. Enter class="custom-class" in the form attributes field
![image](https://github.com/mautic/mautic/assets/706997/f1faeffb-9d0d-4007-9495-ea61d9b219a4)

6. Select Remain at form for the Successful Submit Action field
![image](https://github.com/mautic/mautic/assets/706997/37ad7adf-68f5-4f88-858e-e05bcf46a186)

7. Click on Save & Close
8. Preview the form
![image](https://github.com/mautic/mautic/assets/706997/3c11fb16-e32d-4f53-b599-309ee4adecb2)

9. Right click the submit button and then click on Inspect
![image](https://github.com/mautic/mautic/assets/706997/f29a8702-b8cb-41ea-90c7-fcded3a29c83)

10. From the Elements tab in the browser Developer Tools look for the parent form element and then look for the form's class attribute.
![mautic_form_attr_bug](https://github.com/mautic/mautic/assets/706997/0b75ad5d-9f72-4771-a461-879f334c6f3f)

11. Notice the double quotes "" "" in the class attribute.

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by juangadiel** [comment]
I'm so sorry, I just noticed this is a duplicate of #13645

**Comment by escopecz** [comment]
Thanks for looking for an existing issue! I'll close this one and let's track it in https://github.com/mautic/mautic/issues/13645
